tests/unit_tests/dft/include/compute_inplace_real_real.hpp

```diff
@@ -44,7 +45,14 @@ int DFT_Test<precision, domain>::test_in_place_real_real_USM() {
         descriptor.set_value(oneapi::math::dft::config_param::BWD_DISTANCE, forward_elements);
         descriptor.set_value(oneapi::math::dft::config_param::BACKWARD_SCALE, backward_scale);
 
-        commit_descriptor(descriptor, sycl_queue);
+        try {
+            commit_descriptor(descriptor, sycl_queue);
+        }
+        catch (...) { //BACKWARD_SCALE not supported, go with 1 and scale at check
```

Comment:
commit can throw several exceptions including the one about scaling not supported so I would recommend checking the exception being caught here to make sure it was indeed because of the scaling and not something else.